### PR TITLE
Set plain/text Content-Type for prometheus exporter

### PR DESCRIPTION
**Pull request by snaggen**
Playing around with different tools to view metrics, I found that some tool complained about not having Content-Type text/plain. So this PR fixes that.

### CODE REVIEW

File: metrics-exporter-prometheus/src/exporter/http_listener.rs [hunk 1]
```diff
@@ -69,10 +70,12 @@ impl HttpListeningExporter {
         req: &Request<Incoming>,
     ) -> Response<Full<Bytes>> {
         if is_allowed {
-            Response::new(match req.uri().path() {
+            let mut response = Response::new(match req.uri().path() {
                 "/health" => "OK".into(),
                 _ => handle.render().into(),
-            })
+            });
+            response.headers_mut().append(CONTENT_TYPE, HeaderValue::from_static("text/plain"));
```
Comment (sgrebnov) [inline metrics-exporter-prometheus/src/exporter/http_listener.rs:77]: Should this be `text/plain; version=0.0.4`?
https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format

**Comment by sgrebnov** [comment]
Encountered with the same issue when using this crate and Datadog

**Comment by sgrebnov** [comment]
I've manually applied this fix and confirmed that the Prometheus exporter is now compliant with the DataDog Prometheus scraper.

**Comment by tobz** [comment]
Seems reasonable to me! 👍🏻

**Review by tobz — APPROVED** [review]

**Comment by tobz** [comment]
Released as `metrics-exporter-prometheus@v0.15.3`.

Thanks again for your contribution!

**Pull request merged**